### DISCUSSION TITLE
I've merged two github accounts but old account's commits does not show in new account's contribution graph.

### DISCUSSION THREAD

**Discussion by iizukasan** [community/community#132604] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I've merged two github accounts but old account's commits does not show in new account's contribution graph.
I've just done according to [the article](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts).

I was using my old account for work. So I was working on an Organization's private repository.
Is it the reason? If I was using the organization's private repository, Can't I merge commits to new account?

※ I pushed some branch and created pull request after merging and I could see the new account's contribution graph was updated, but still does not show old commits...

**Comment by Surajk7725** [comment]
@iizukasan There might be some reasons:

1)GitHub tracks contributions based on email addresses used in commits.(Add your old account email in your new email)
2)Ensure that contributions from private repositories are enabled to show on your contribution graph.(Go to Profile Settings -> Check the box that says "Include private contributions on my profile".)
3)Also check your repository access.

Last solution will be using "git filter-branch"

**Reply by iizukasan** [reply]
@Surajk7725 
Thank you for your comment!
I've done 1 and 2. (I did that like 3-4 days ago so it's not because of reflection delay).
About 3, do you mean if I can access to the private repository (in which I had commits with the old account) with the new account?

"git filter-branch" changes all of my past commits so I don't want to do that...

**Comment by iizukasan** [comment]
as time passed, it's reflected.
